src/TodoList.js

```diff
@@ -1,15 +1,14 @@
 import React from "react";
+import TodoListItem from "./TodoListItem";
 const todoList =[{id: 1, tittle: 'Grocery shopping'}, {id: 2, tittle:'pack boxes'}, {id:3, tittle:'buy headphones'}];
 
 function TodoList() {
   return (
     <div>
     <ul>
-      {todoList.map((el)=>{
```

Comment:
same, you can remove a <div> element, just return a <ul>

Reply:
I don't exactly understand what you mean.  Do you mean remove the div and only return de ul?

Reply:
yes,sorry, just return an ul